README.md

```diff
@@ -0,0 +1,25 @@
+Vimeo XBlock
+============
+
+This XBlock allows course creators to stream videos from [Vimeo](https://vimeo.com/) from within their courses.
+
+Install
+-------
+
+See [edX Installing an XBlock](http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/install_xblock.html?highlight=install%20xblock)
+for details on how to install this XBlock on your Open edX instance.
+
+```python
+pip install "git+https://github.com/xblock-vimeo/xblock-vimeo@master#egg=xblock-vimeo"
```

Comment:
@kaizoku This URL needs to use our repo URL.  Should be `git+https://github.com/open-craft/xblock-vimeo@master#egg=xblock-vimeo`